From a4353d936fe205471e8911a5c9652874cb3a2c71 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Sun, 20 Aug 2023 14:17:44 +0200
Subject: [PATCH] board: solidrun: imx8mp: remove dtb fixup for hummingboard
 extended

Now that a seperate dtb file is available for HB extended, no need for
runtime patching.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 .../solidrun/imx8mp_solidrun/imx8mp_solidrun.c | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/board/solidrun/imx8mp_solidrun/imx8mp_solidrun.c b/board/solidrun/imx8mp_solidrun/imx8mp_solidrun.c
index 9e5748ab2b5..adef8c0c3a0 100644
--- a/board/solidrun/imx8mp_solidrun/imx8mp_solidrun.c
+++ b/board/solidrun/imx8mp_solidrun/imx8mp_solidrun.c
@@ -260,24 +260,6 @@ int ft_board_setup(void *blob, struct bd_info *bd)
 	}
 #endif
 
-	if (strcmp(board_id.carrier_name, "hummingboard-extended") == 0) {
-		pr_info("Disabling M.2 rfkill for hummingboard extended \n");
-		char *path = "/rfkill_m2_wdis";
-    	int nodeoffset = fdt_path_offset(blob, path);
-    	if (nodeoffset < 0) {
-        	printf("Unable to find node %s\n", path);
-        	return -1;
-    	}
-
-		const char *name = "status";
-    	const char *val = "disabled";
-		int rc = fdt_setprop_string(blob, nodeoffset, name, val);
-		if (rc < 0) {
-			printf("Unable to set property %s to %s\n", name, val);
-			return -1;
-		}
-	}
-
 	return 0;
 }
 
